### PR TITLE
Improve api

### PR DESCRIPTION
**Pull request by jroosing**

### CODE REVIEW

File: cmd/kiya/cmd_move.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ func commandMove(
 		if err := kiya.Move(kmsService, storageService, source, sourceKey, target, targetKey); err != nil {
 			log.Fatal(err)
 		}
+		fmt.Printf("Successfully copied [%s] to [%s]\n", sourceKey, target.Label)
```
Comment (emicklei) [inline cmd/kiya/cmd_move.go]: copied or moved?

File: gcp_storage.go [hunk 2]
```diff
@@ -18,7 +18,10 @@ func StoreSecret(storageService *cloudstore.Client, target Profile, key, encrypt
 	w := bucket.Object(key).NewWriter(context.Background())
 	defer w.Close()
 	_, err := fmt.Fprintf(w, encryptedValue)
-	return tre.New(err, "writing encrypted value failed", "encryptedValue", encryptedValue)
+	if err != nil {
```
Comment (emicklei) [inline gcp_storage.go]: no need to do nil test

File: kiya.go [hunk 3]
```diff
@@ -56,9 +55,8 @@ func Move(kmsService *cloudkms.Service,
 	if err := PutSecret(kmsService, storageService, target, targetKey, sourceValue); err != nil {
 		return tre.New(err, "save key failed", targetKey)
 	}
-	fmt.Printf("Successfully copied [%s] to [%s]\n", sourceKey, target.Label)
 	// delete key from source
-	if err := DeleteSecret(kmsService, storageService, target, targetKey); err != nil {
+	if err := DeleteSecret(kmsService, storageService, source, sourceKey); err != nil {
```
Comment (emicklei) [inline kiya.go]: strange change, target->source.   r u sure?

File: cmd/kiya/cmd_move.go (hunk 1 above)
Comment (amohabir) [reply cmd/kiya/cmd_move.go]: Also use log package over fmt package?

File: kiya.go (hunk 3 above)
Comment (jroosing) [reply kiya.go]: Given that this is in the "move" command, we fist "put" the secret in the new space (e.g. copy), then we need to delete the origin (source), making this a "move"

**Pull request merged**